From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sun, 17 Oct 2021 22:53:22 -0400
Subject: [PATCH] SystemUI: Use new privacy indicators for location

Android has had location indicators for a while, but let's use the new
privacy indicator infrastructure for displaying them. This makes them
integrate better with the new camera and microphone indicators.
---
 .../permissioncontroller/permission/utils/KotlinUtils.kt        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PermissionController/src/com/android/permissioncontroller/permission/utils/KotlinUtils.kt b/PermissionController/src/com/android/permissioncontroller/permission/utils/KotlinUtils.kt
index 3fcc88854..c87dde293 100644
--- a/PermissionController/src/com/android/permissioncontroller/permission/utils/KotlinUtils.kt
+++ b/PermissionController/src/com/android/permissioncontroller/permission/utils/KotlinUtils.kt
@@ -197,7 +197,7 @@ object KotlinUtils {
             DeviceConfig.getBoolean(
                 DeviceConfig.NAMESPACE_PRIVACY,
                 PROPERTY_LOCATION_INDICATORS_ENABLED,
-                false
+                true
             )
     }
 
